doc/UsersGuide/source/Backgroundinfo/TechnicalOverview.rst

```diff
@@ -91,3 +91,151 @@ HAFS Workflow Schematic
+
+**Developed by:**
+    EMC and the UFS Hurricane Application Team. See currently active HAFS developers `here <https://docs.google.com/presentation/d/1xBNxvAG8-Kk3GS93PndaPVZp_L8U-KqGRGNhnviXZMg/edit?usp=sharing>`__
+    
+**Runs on:** WCOSS2
+
+Input/Output
+------------
+
+**Input:**
+    - Global Forecast System (GFS, including GFS/GDAS) NetCDF format analysis for atmospheric initial condition, and 3-hourly GRIB2 files (up to 129 h) for atmospheric lateral boundary conditions and atmospheric forcing for oceanic and wave model components.
+    - Global RTOFS for oceanic initial and boundary conditions for MOM6 and HYCOM.
+    - GFS/GDAS global wave analysis/forecast for HFSA wave component initial and lateral boundary conditions.
+    - OBSPROC observational data for HAFS data assimilation.
+
+**Output:**
+    - ATCF track forecast files, 4x/day at 00Z/06Z/12Z/18Z.
+
+Locations for Output
+--------------------
+
+- **WCOSS2:**
+    - HFSA: ``/lfs/h1/ops/prod/com/hafs/v2.0/hfsa.yyyymmdd/hh``
+    - HFSB: ``/lfs/h1/ops/prod/com/hafs/v2.0/hfsb.yyyymmdd/hh``
+- **Other servers:**
+    - NCEP ftp server, NOMADS, NWS Gateway/SBN
+
+Background
+----------
+
+Hurricane Analysis and Forecast System (HAFS), as the Unified Forecast System (UFS) hurricane application, is the FV3 (Finite­ Volume Cubed-Sphere Dynamical Core) based multi-scale model and data assimilation system capable of providing tropical cyclone (TC, including hurricane and typhoon) analyses and forecasts of the inner core structure key to improving storm size and intensity predictions, as well as the large-scale environment that is known to influence the TC's motion. HAFS development targets an operational data assimilation and modeling system, capable of providing reliable, robust and skillful model guidances for TC track, intensity (including rapid intensification), size, structure, rainfall and cyclone genesis, as well as for TC associated storm surge, sea surface waves, and tornadoes. HAFS is a community-based coupled earth modeling system specially calibrated for hurricane prediction with TC dynamics and physics, sophisticated vortex initialization and advanced inner-core data assimilation techniques, and various air-sea interaction processes.
+
+HAFSv2 will replace NCEP's current operational hurricane forecast systems, HAFSv1 in the 2024 hurricane season. The goal of this project is to upgrade the HAFSv1 that was running in operations in 2023. The system will provide improved and comparable tropical cyclone track and intensity forecast guidance in all global oceanic basins, as demonstrated through evaluation of retrospective and real time experiments compared against the two current configurations of operational HFSAv1 and HFSBv1.
```

Comment:
Most of this information is already included in the introduction. I would remove this and update the introduction section with any details that appear here but not there. In the introduction, you can then link to the section below. For example: 
"The specific HAFSv2 scientific changes to improve track and intensity forecast skills are summarized in :numref:`Section %s <scientific-updates>`."